### DISCUSSION TITLE
how to ues ekuiper tflite demo

### DISCUSSION THREAD

**Discussion by keven808** [lf-edge/ekuiper#1898] [Q&A]
I learned how to use video detection in the example on the ekuiper official website, but I have an error in the resize function during use, how to solve it?
![Uploading Q~NUW9]~A`([S9)71[3[F5U.png…]()

**Comment by keven808** [comment]
![Q~NUW9 ~A`( S9)71 3 F5U](https://github.com/lf-edge/ekuiper/assets/133673052/4f2b3ce2-14fb-4171-8086-0b5f2750a279)

**Reply by superrxan** [reply]
What's the problem look like ? could you please give us more infomation

**Comment by ngjaying** [comment]
@keven808  As discussed offline, you have solved the problem. Would you please update how/what did you find? Thanks.
